### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by ThePixelDeveloper**
pls

### CODE REVIEW

**Comment by ThePixelDeveloper** [comment]
@meadsteve pls

File: src/SplFix/LimitIterator.php [hunk 1]
```diff
@@ -3,23 +3,41 @@
...
+class LimitIterator extends \LimitIterator implements Countable
 {
-
+    /**
+     * @var integer
+     */
     protected $maxCount;
 
+    /**
+     * LimitIterator constructor.
```
Comment (meadsteve) [inline src/SplFix/LimitIterator.php:21]: nope.

File: src/SplFix/LimitIterator.php [hunk 2]
```diff
@@ -3,23 +3,41 @@
...
 use Iterator;
+use Countable;
 
-class LimitIterator extends \LimitIterator implements \Countable
+/**
+ * Class LimitIterator
+ *
+ * @package MeadSteve\SplFix
+ */
+class LimitIterator extends \LimitIterator implements Countable
```
Comment (meadsteve) [inline src/SplFix/LimitIterator.php:13]: :trollface:

**Comment by meadsteve** [comment]
#burn

**Pull request closed without merging**